src/templates.rs

```diff
@@ -104,7 +104,9 @@ pub enum FormError {
     /// The password and it's repeat are not the same
     PasswordMismatch,
     /// The email is already in use by another user
-    UserExists,
+    EmailExists,
+    /// The github handel is already in use by another user
+    GitExists,
```

Comment:
Change this to `GithubExists` so it's clearer that we're talking about GitHub since we might want to add other providers like GitLab later.